frontend/src/components/UI/Modal/Modal.jsx

```diff
@@ -70,7 +71,14 @@ export const Modal = ({
                                 >
                                     {heading}
                                 </h2>
-                                {description && (
+                                {description && description instanceof Array && (
```

Comment:
I am seeing this warning in terminal

Failed prop type: Invalid prop `description` of type `array` supplied to `Modal`, expected `string`.